components/view_image/popover_bar/popover_bar.tsx

```diff
@@ -96,16 +99,17 @@ export default class PopoverBar extends React.PureComponent {
             );
         }
 
-        let zoomControls = [];
+        const zoomControls: React.ReactNode[] = [];
+        let wrappedZoomControls: React.ReactNode = null;
```

Comment:
previous code was as follows, and TypeScript complained about assigning a React Node to array:
1. initialize `zoomControls` with empty array
2. push some react nodes to it 
3. wrap`zoomControls` with div and assign it to `zoomControls` itself

so I created new variable `wrappedZoomControls` to make it simpler

Reply:
IMO the `wrappedZoomControls` is not needed. It is generally better to avoid such constructs, that is true, but instantiating a new variable just to wrap elements, is a bit much.

Could you please move the wrapping `<div>` to the actual rendering?

Reply:
Agreed, just need to make sure if the rendering of the empty <div> doesn't add any changes to the styling

Reply:
i see, i can move the `<div>` to actual rendering. 
but in your suggestion, we render the div when `this.props.showZoomControls == true` as well, which is different from master's logic. is that OK? or should I do conditional rendering as I wrote in suggestion?